content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,146 @@ schema: olm.channel
+```cue
+#CompositeConfig: {
+	Schema: "olm.composite"
+	Components: [...#Component]
+}
+
+#Component: {
+	Name:        string
+	Destination: #ComponentDestination
+	Strategy:  #BuildStrategy
+}
+
+#ComponentDestination {
+	Path: string
+}
+
+#BuildStrategy: {
+	Name:   string
+	Template: #TemplateDefinition
+}
+
+#TemplateDefinition: {
+	Schema: string
+}
+```
+
+#### Catalog Maintainer
+Representing the catalog maintainer is the `olm.composite.catalogs` schema which defines:
+- One or more catalogs (`Catalog`) identified by a string (`Catalog.Name`)
+- Supported input formats (`Builder`) for each `Catalog` (`Catalog.Builders`) which is a list of strings where each item is a `Builder` schema. The currently supported `Builder` schemas are:
+    - `olm.builder.basic` which represents that the use of [basic templates](#basic-template) for catalog contribution generation is allowed
+    - `olm.builder.semver` which represents that the use of [semver templates](#semver-template) for catalog contribution generation is allowed
+    - `olm.builder.raw` which represents that the use of raw FBC for catalog contribution generation is allowed
+    - `olm.builder.custom` which represents that the use of custom catalog contribution generation logic is allowed
+- The expected directory structure of any generated contribution (`Catalog.Destination.WorkingDir`)
+
+The cue schema for the `olm.composite.catalogs` schema is:
+```cue
+#CatalogConfig: {
+	Schema: "olm.composite.catalogs"
+	Catalogs: [... #Catalog]
+}
+
+#Catalog: {
+	Name:        string
+	Destination: #CatalogDestination
+	Builders: [...string]
+}
+
+#CatalogDestination: {
+	BaseImage:  string
+	WorkingDir: string
+}
+```
+
+### Examples
+**catalogs.yaml**
+
+The following example specifies a catalog configuration that defines a catalog named `v1` and the contribution generation logic to only accepting input that uses the [basic template](#basic-template). When using the `opm alpha render-template composite` subcommand the path to this file can be specified with the `-f` option and can be a file path or a URL (the URL must return the raw file contents).
+
+```yaml
+schema: olm.composite.catalogs
+catalogs:
+- name: v1 {{< code_callout 1 >}}
+  destination:
+    baseImage: quay.io/operator-framework/opm:v1.24
+    workingDir: catalogs/v1 {{< code_callout 2 >}}
+  builders:
+    - olm.builder.basic {{< code_callout 3 >}}
+```
+
+- {{< code_callout 1 >}} Defines the `v1` catalog configuration
+- {{< code_callout 2 >}} Defines the directory path that all output should be placed for this catalog. In this example, when rendering using the composite catalog all output for the `v1` catalog will be put under the `catalogs/v1/` directory
+- {{< code_callout 3 >}} Defines the allowed builders for the `v1` catalog. In this example the only allowed builder is the basic builder, meaning the only contributions allowed to be rendered for this catalog are basic templates
```

Comment:
```suggestion
- {{< code_callout 3 >}} Defines the allowed builders for the `v1` catalog. In this example, the basic builder is the only allowed builder. As a result, this catalog renders allows only basic templates as contributions.
```
Trying to simplify, but maybe I lost the thread. Feel free to disregard if I am off base.

Reply:
Addressed in https://github.com/operator-framework/olm-docs/pull/286/commits/1cdb364fe5c9f65cbc47ff7effdb3c41bf104cc0

Reply:
/resolve